notebooks/iceflow/iceflow/layers.py

```diff
@@ -14,14 +14,17 @@
                           max_zoom=4)
 
 df_north = geopandas.read_file('./iceflow/files/ib_north.json')
-df_north['timestamp'] = df_north['timestamp'].str[:-6]
+df_north['timestamp'] = df_north['timestamp'].dt.tz_convert(None).astype(str)
```

Comment:
I still received the same `TypeError: <class 'bytes'> is not convertible to datetime` error when running the Pandas lines again in the iceflow notebooks (I wrote up CRYO-86 separately)